style-guide.md

```diff
@@ -8,10 +8,10 @@ Use this file for language-specific style rules to follow for translation.
 
 Leave text in code blocks untranslated except for comments. You may optionally translate text in strings, but be careful not to translate strings that refer to code!
 
-Example:
+Exemple:
 
 ```js
-// Example
+// Exemple
 import React from "react"
 export default () => (
   <div style={{ color: `purple`, fontSize: `72px` }}>Hello Gatsby!</div>
```

Comment:
Why all the example are translated and not this one?

Reply:
Oops

Reply:
This one is a false positive. Don't make the changes